src/assisted_installer_controller/assisted_installer_controller.go

```diff
@@ -57,6 +57,7 @@ func (c *controller) WaitAndUpdateNodesStatus() {
 	assistedInstallerNodesMap := c.getInventoryNodesMap()
 	for len(assistedInstallerNodesMap) > 0 {
 		time.Sleep(GeneralWaitTimeout)
+		c.log.Infof("Searching for host to change status")
```

Comment:
lets add a log also on line 68, since we would want to know that there is a node in kubernetes , but not in bm-inventory. it will be a rare occasion, so let's catch it

Reply:
not a good idea. cause we are deleting host from map. so u will always have hosts that are not part of it

Reply:
as discussed, will move this issue to another ticket